### PR TITLE
Fix issue with HostedCluster APIServer definition in hypershift-agent-template

### PR DESCRIPTION
**Pull request by batzionb**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -3,60 +3,60 @@ entries:
...
     apiVersion: v2
-    created: "2023-06-15T12:57:53.103862936+02:00"
+    created: "2023-07-17T11:40:30.08928852+03:00"
     description: A Helm chart which installs Hypershift cluster using KubeVirt platform
     digest: 2b5d44df585930cc6b4c4129937b0434e8ebc957d08f6d084ae8d501f1dcabcb
     name: hypershift-kubevirt-template
     type: application
     urls:
-    - https://stolostron.github.io/cluster-templates-manifests/hypershift-kubevirt-template-0.0.3.tgz
+    - https://batzionb.github.io/cluster-templates-manifests/hypershift-kubevirt-template-0.0.3.tgz
```
Comment (machacekondra) [inline index.yaml]: can you please move back to `stolostron`?

Comment (batzionb) [reply index.yaml]: I removed the changes in the index.yaml all together
seems like it doesn't need reindexing since there's no version added

**Review by machacekondra — APPROVED** [review]
/lgtm

**Pull request merged**